kolibri/core/discovery/migrations/0002_auto_20180825_1744.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
since we haven't merged this, could probably just make a single migration file for now.